readme-template.md

```diff
@@ -0,0 +1,92 @@
+# Repo Title
+
+<!-- Not all of these are essential but keep this in mind for the repo you are using
+![Build badge]() ![Test Coverage]() ![npm package]() -->
+
+<!--Short sentence on what the repo does-->
+
+##Contents
+* [Further Documentation](#further-documentation)
+* [System Overview](#system-overview)
+* [Installation](#installation)
+* [Running Locally](#running-locally)
+* [Testing](#testing)
+	* [Automated Testing](#automated)
+	* [Manual testing](#manual)
+* [Script cheat sheet](#cheat-sheet)
+* [Built With](#built-with)
+* [Linting](#linting)
+* [Deploying](#deploying)
+* [Contributing Rules](#contributing-rules)
+* [Common Questions](#common-questions)
+
+## Further Documentation
+
+<!-- Examples of some documentation to link from the readme -->
+* **[System Roadmap]()**
+* **[Regression Testing Checklist]()**
+* **[Change Log]()**
+
+Include any other relevant documentation in this section.
+
+## System Overview
+
+* More in-depth overview of the system, what this is trying to accomplish and how it achieves its goal.
+* Keep it concise, write this section like the first paragraph of a blog post, what is the point of this repo?
+* Perhaps label some of the core tech used in the repo and link to the external websites/documentation
+
+## Installation
+
+* Step-by-step guide on installation
+
+## Running Locally
+
+* Step-by-step guide of setting up locally
+
+## Script Cheat Sheet
+
+* List scripts used in the repo and a description of what they do
+* Example: `npm run build` := Generates `dist` folder with changes made in `lib`
+
+## Testing
+
+* What CI do the test run on?
+* What testing framework is used?
+* How are the tests structured in this repo? Is there a strict writing style to follow?
```

Comment:
should that be in here - do we have any repos which are very different for this? i'd prefer to have a general convention for this (in the tech / test best practices docs)